File: KSTU_studying/c++_basis/lab_2/python/ex02.py
arr = [-1, 3, -5, 6, -7, -8, 10, 10]

for i in range(len(arr) - 1):
    if(arr[i] > 0):
        if(arr[i + 1] > 0):
            print(arr[i], arr[i + 1])
            break
    if(arr[i] < 0):
        if(arr[i + 1] < 0):
            print(arr[i], arr[i + 1])
            break

File: KSTU_studying/c++_basis/lab_2/c++/ex02.cpp
#include <iostream>
#include <vector>

int main(void) {
    int array_count = 10;
    std::vector<int>arr(array_count);
    for (int i = 0; i < array_count; i++) {
        arr[i] = rand() % 11 - 5;
    }
    //полный список
    std::cout << "full array:" << "\n";
    for (int i = 0; i < array_count; i++) {
        std::cout << "[" << arr[i] << "]";
    }
    //ближайшие числа с одинаковым знаком
    std::cout << "\n" << "Numbers with the same sign:" << "\n";
    for (int i = 0; i < array_count - 1; i++) {
        if(arr[i] * arr[i + 1] > 0) {
            std::cout << "[" << arr[i] << "]" << "[" << arr[i + 1] << "]";
            break;
        }
    }
}